### ISSUE TITLE
Add iris middleware

### ISSUE THREAD

**Issue by eabili0** [labbsr0x/whisper-client#6]

**Comment by claudiosegala** [comment]
@abilioesteves you want the same that is implemented in `GetMuxSecurityMiddleware` but with Iris?
https://github.com/labbsr0x/whisper-client/blob/faaa25fe15dde36f7c8cbb9f60cb87752211dc60/client/client.go#L87-L107

**Comment by eabili0** [comment]
Yes, very similiar implementation, but for Iris @claudiosegala .
